src/pages/Vaults/Vault/UpdateCollateralModal/index.tsx

```diff
@@ -266,15 +268,34 @@ const UpdateCollateralModal = ({
     }
   };
 
-  const renderRequiredCollateralTokenAmount = () => {
-    if (requiredCollateralTokenAmountIdle || requiredCollateralTokenAmountLoading) {
+  const renderMinRequiredCollateralTokenAmount = () => {
```

Comment:
minor issue, but I'm finding these function names odd (`renderMinRequiredCollateralTokenAmount` and ` renderMaxWithdrawableCollateralTokenAmount`) - `render` implies an element being returned rather than a value. Could they be changed to 'getMinRequiredCollateralTokenAmount` and `getrenderMaxWithdrawableCollateralTokenAmount` or similar?

Reply:
Done!